homework/shelkovich_ilya/lookUpArrayService/src/test/java/com/tutrit/java/quickstart/service/LookUpArrayServiceTest.java

```diff
@@ -0,0 +1,60 @@
+package com.tutrit.java.quickstart.service;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class LookUpArrayServiceTest {
+
+    private LookUpArrayService lookUpArrayService;
+
+    @Before
+    public void setUp() {
+        this.lookUpArrayService = new LookUpArrayService();
+    }
+
+    @Test
+    public void findMax() {
+        int[] array = {1, 3, 12, 4};
+        Assert.assertEquals(12, lookUpArrayService.findMax(array));
+    }
+
+    @Test
+    public void findMin() {
+        int[] array = {1, 3, 12, 4};
+        Assert.assertEquals(1, lookUpArrayService.findMin(array));
+    }
+
+    @Test
+    public void shouldReturn0IfArrayIsEmpty() {
+        int[] emptyArray = {};
+        Assert.assertEquals(0, lookUpArrayService.findMax(emptyArray));
+        Assert.assertEquals(0, lookUpArrayService.findMin(emptyArray));
```

Comment:
I guess one of the asserts redundant.

Reply:
@max-java 
Here he test two methods. I think it is not redundant.

@max-java 
So can you write tests? Do write test method with new title?
In class LookUpArrayService is not method "shouldReturn0IfArrayIsEmpty"....
 @Test
    public void shouldReturn0IfArrayIsEmpty()

Reply:
deleted one assertion

Reply:
replied in team chat with best practices link